trigger-openqa_in_openqa

```diff
@@ -56,6 +56,27 @@ download_scenario() {
     curl https://raw.githubusercontent.com/os-autoinst/os-autoinst-distri-openQA/master/scenario-definitions.yaml -o /var/tmp/sd.yaml
 }
 
+check_package_state() {
+    local package=$1
+    local state
+    state=$($osc results -r openSUSE_Tumbleweed -a x86_64 --no-multibuild "$src_project" "$package")
+
+    case "$state" in
+        *succeeded* | *disabled*)
+            return 0
+            ;;
+        *)
+            return 1
+            ;;
+    esac
```

Comment:
So you are replacing a grep call with bash internal equality match but which unfortunately leads to the rather clunky multi-line case-statement. If you want to use bash then you could look into the bash pattern match operator, something like

```
[[ "$state" =~ 'succeeded\|disabled' ]]
```

but in the original place. Note that this didn't immediately work for me. I would just keep the current state as is